app/controllers/api/videos_controller.rb

```diff
@@ -82,10 +82,10 @@ def next_suggestions
   end
 
   def show
-    @video = Video.includes(:channel, :comments, :views).find(params[:id])
+    @video = Video.includes(:channel, :views, comments: [:replies, likes: [:user]]).find(params[:id])
     if @video
-      @comments = @video.comments
-      @channel = @video.channel;
+      @comments = @video.comments.where({parent_comment_id: nil})
```

Comment:
remove curly braces in where clause, where takes a hash, new Ruby we don't need to specify that its a Hash if its key value pair

Reply:
awesome